spec/web_helpers.rb

```diff
@@ -0,0 +1,28 @@
+module Helpers
+
+  def sign_up(email, password)
+    visit('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.